src/runtime/cli/main.go

```diff
@@ -120,27 +120,11 @@ var runtimeFlags = []cli.Flag{
 // runtimeCommands is the list of supported command-line (sub-)
 // commands.
 var runtimeCommands = []cli.Command{
-	createCLICommand,
-	deleteCLICommand,
-	execCLICommand,
-	killCLICommand,
-	listCLICommand,
-	pauseCLICommand,
-	psCLICommand,
-	resumeCLICommand,
-	runCLICommand,
-	specCLICommand,
-	startCLICommand,
-	stateCLICommand,
-	updateCLICommand,
-	eventsCLICommand,
 	versionCLICommand,
 
 	// Kata Containers specific extensions
 	kataCheckCLICommand,
 	kataEnvCLICommand,
-	kataNetworkCLICommand,
-	kataOverheadCLICommand,
```

Comment:
Are these two still needed? `network` provides possibility to implement enlightened CNI and `overhead` collects pod overhead to be useful for kubernetes `PodOverhead` feature.

Reply:
Subcommand `kata-network` and `kata-overhead` are not working with containers started by shim v2.

`list` worked because it's get sandboxes from persist store, but `kata-network` and `kata-overhead` are using `ContainerIDMapping` stored under `/var/run/kata-containers/containers-mapping`.

```
# ./kata-runtime  list
ID                                                                 PID         STATUS      BUNDLE                                                                                                                  CREATED                          OWNER
d86a4fdd07bc1f2d165717d18f1c25284ebf384c8ebef5156a5064e476bb78aa   -1          running     /run/containerd/io.containerd.runtime.v2.task/k8s.io/d86a4fdd07bc1f2d165717d18f1c25284ebf384c8ebef5156a5064e476bb78aa   2020-07-07T02:23:21.744302224Z   #0
a40d9349eea6d61545d006f84db631211d5c44bce0ec05fcfe8918adba1cb3c8   -1          running     /run/containerd/io.containerd.runtime.v2.task/k8s.io/a40d9349eea6d61545d006f84db631211d5c44bce0ec05fcfe8918adba1cb3c8   2020-07-07T02:23:54.267641529Z   #0

# ./kata-runtime  kata-overhead d86a4fdd07bc1f2d165717d18f1c25284ebf384c8ebef5156a5064e476bb78aa
Container ID (d86a4fdd07bc1f2d165717d18f1c25284ebf384c8ebef5156a5064e476bb78aa) does not exist
# ./kata-runtime  kata-overhead a40d9349eea6d61545d006f84db631211d5c44bce0ec05fcfe8918adba1cb3c8
Container ID (a40d9349eea6d61545d006f84db631211d5c44bce0ec05fcfe8918adba1cb3c8) does not exist

# ./kata-runtime  kata-network  list-ifaces a40d9349eea6d61545d006f84db631211d5c44bce0ec05fcfe8918adba1cb3c8
Container ID (a40d9349eea6d61545d006f84db631211d5c44bce0ec05fcfe8918adba1cb3c8) does not exist
# ./kata-runtime  kata-network  list-ifaces d86a4fdd07bc1f2d165717d18f1c25284ebf384c8ebef5156a5064e476bb78aa
Container ID (d86a4fdd07bc1f2d165717d18f1c25284ebf384c8ebef5156a5064e476bb78aa) does not exist
```

For `kata-overhead`, which is a help command but not directly called from K8s, we can re-implement it in shim v2. 

But I have no ideas if the `kata-network` is needed anymore.

Reply:
And `kata-overhead` works only when `sandbox_cgroup_only=true`

Reply:
Sounds like that they should be re-implemented on top of shimv2? And we probably should add a kata admin API socket that can expose more functionalities like network interface management. WDYT? /cc @amshinde @egernst

Reply:
I'm working on getting pod overhead from metrics: https://github.com/liubin/kata-containers/commit/87e72349b5d23d3823a367cd0d3601a5712ef93a

Reply:
pod overhead part is merged. Let's merge this one too! The network configuration part might still need extra discussion but should not block removal of the malfunctioning old implementation.